src/components/browse/datasetlist/datasetheader/datasetdetails/datasetDetails.tsx

```diff
@@ -40,38 +46,95 @@ const DatasetDetails = (props: dataSetDetailsProps) => {
-              <span style={{ userSelect: "all" }}>{props.hit.content.title}</span>
+              <span style={{ userSelect: "all" }}>
+                {props.hit.content.title}
+              </span>
             </p>
           </Col>
           <Col lg md sm xl xs xxl="1" className="text-end px-0">
-            {props.details !== null && props.details !== undefined ?
-              <Button className="fs-8 w-100"
-                onClick={() => handleOpen()}>Request Access</Button> :
-              <Button className="fs-8 w-100" disabled
-                onClick={() => handleOpen()}>Request Access</Button>
-            }
+            {props.details !== null && props.details !== undefined ? (
+              <Button className="fs-8 w-100" onClick={() => handleOpen()}>
+                Request Access
+              </Button>
+            ) : (
+              <Button
+                className="fs-8 w-100"
+                disabled
+                onClick={() => handleOpen()}
+              >
+                Request Access
+              </Button>
+            )}
           </Col>
-          <Modal size="lg" centered
-            show={show} onHide={handleClose}>
-            <Modal.Header closeButton>
-              <Modal.Title>Request access for dataset {props.hit.content.accession}</Modal.Title>
+          <Modal size="lg" centered show={show} onHide={handleClose}>
+            <Modal.Header closeButton className="border-0">
+              <Modal.Title>
+                <FontAwesomeIcon
+                  icon={faDownload}
+                  className="text-muted me-3"
+                />
+                <strong>
+                  How to request access for dataset{" "}
+                  {props.hit.content.accession}
+                </strong>
+              </Modal.Title>
             </Modal.Header>
 
-            <Modal.Body>
-              <p>Please copy the following message and send us a mail to {mailId}. If configured, the
-                message opens in your email client. Please provide any additional details if needed.
-              </p>
-              <hr />
-              <p>Hello DAC team,
-                <br /><br />
-                Since I am interested in the topic {props.hit.content.title}, I would like to request access to the Dataset {props.hit.content.accession}.
-                Kindly grant the access for the requested dataset.
-                <br /><br />
+            <Modal.Body className="px-4">
+              <Row className="mb-3 p-3 bg-warning align-items-center">
+                <Col lg md sm xl xs xxl={1}>
+                  <FontAwesomeIcon
+                    icon={faCircleExclamation}
+                    className="text-danger"
+                    size="2x"
+                  />
+                </Col>
+                <Col>
+                  Please copy the following message and send us a mail to{" "}
+                  <a href={"mailto:" + mailId}>{mailId}</a>. If configured, the
```

Comment:
It looks like a link in the wireframes, but what does this link point to? Opening the mail client without any message filled in might not be a desirable feature. 

My take is, we dont to have make the mail id a "link" since we have a button in the popup. Maybe the font color for the link can be changed to link just to reflect the wireframes. 

If the design team do need a link here then we can check if they want an empty body mail client opened or should the link serve the same purpose as the button.